tracer/src/Datadog.Trace/ClrProfiler/Instrumentation.cs

```diff
@@ -371,7 +373,7 @@ private static void InitRemoteConfigurationManagement(Tracer tracer)
                         var rcmApi = RemoteConfigurationApiFactory.Create(tracer.Settings.Exporter, rcmSettings, discoveryService);
                         var tags = GetTags(tracer, rcmSettings);
```

Comment:
Probably need to add the normalization inside `GetTags` too. For example, we add the `env` there, so could end up with a discrepancy with the RCM below

Reply:
Done in 07e24548631653b407159077cf3de015d03b5cf6